components/brave_wallet/resources/solana_provider.js

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+(function () {
+  if (!window.solana) {
+    return
+  }
+  const solanaWeb3 = require('@solana/web3.js');
```

Comment:
i notice there is a semicolon here but no semicolon in other lines. should we be running some kind of js standard linter on these files?

Reply:
I will remove it. our js-lint doesn't run on bundle js files

Reply:
fixed in fd22ab5893343527dd73605f2e040037b2be05a7